### PR TITLE
Ref: .NET troubleshooting.

### PR DESCRIPTION
**Pull request by lucas-zimerman**
- Default SDK troubleshooting moved to include path. (It would be nice if all include paths in regard to the platform information would be grouped in a single folder, just like how it happens with performance and enriching-events)
- Moved ASP.NET SDK  proxy and TLS troubleshooting to the troubleshooting page, incrementing the default troubleshooting page.

### CODE REVIEW

File: src/includes/troubleshooting/dotnet.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Build Issues
+
+
+### Package conflict - `Conflict with Sentry.DiagnosticsSource.`
```
Comment (imatwawana) [inline src/includes/troubleshooting/dotnet.mdx:4]:
```suggestion
### Package Conflict - `Conflict with Sentry.DiagnosticsSource`
```

File: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+title:  Troubleshooting
+sidebar_order: 100
+description: "Learn more about how to troubleshoot common issues with the ASP.NET SDK. "
+---
+
+## Configuration Issues
+
+### Proxy server
+
+Often your server can only access the Internet through a proxy server.
```
Comment (imatwawana) [inline src/platforms/dotnet/guides/aspnet/troubleshooting.mdx:11]:
```suggestion
Often, your server can only access the internet through a proxy server.
```

File: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title:  Troubleshooting
+sidebar_order: 100
+description: "Learn more about how to troubleshoot common issues with the ASP.NET SDK. "
+---
+
+## Configuration Issues
+
+### Proxy server
```
Comment (imatwawana) [inline src/platforms/dotnet/guides/aspnet/troubleshooting.mdx:9]:
```suggestion
### Proxy Server
```

File: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+title:  Troubleshooting
+sidebar_order: 100
+description: "Learn more about how to troubleshoot common issues with the ASP.NET SDK. "
```
Comment (imatwawana) [inline src/platforms/dotnet/guides/aspnet/troubleshooting.mdx:4]:
```suggestion
description: "Learn more about how to troubleshoot common issues with the ASP.NET SDK."
```

File: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+```xml
+<system.net>
+    <defaultProxy>
+      <proxy proxyaddress="http://proxy.address.here" bypassonlocal="true" />
+    </defaultProxy>
+</system.net>
+```
+
+### TLS 1.2 Support and Windows Server
```
Comment (imatwawana) [inline src/platforms/dotnet/guides/aspnet/troubleshooting.mdx:23]:
```suggestion
### TLS 1.2 Support & Windows Server
```

File: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+If you're not able to capture events from ASP.NET to Sentry,
+on older versions of Windows Server, with older .NET Framework you must enable TLS 1.2.
```
Comment (imatwawana) [inline src/platforms/dotnet/guides/aspnet/troubleshooting.mdx:26]:
```suggestion
If you're not able to capture events from ASP.NET to Sentry
on older versions of Windows Server, with the older .NET Framework, you must enable TLS 1.2.
```

**Review by imatwawana — APPROVED** [review]
Minor wording and style edits pending technical review.

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**